### PR TITLE
yet another autorcm change

### PR DESCRIPTION
**Pull request by quadsYT**

### CODE REVIEW

**Comment by Halvorsen256** [comment]
A small tidbit should be in this bullet regarding preservation, as it makes the issue seem more critical than it actually is.

"-When a system update is completed while stock firmware is running, AutoRCM becomes disabled. AutoRCM can be preserved by updating while CFW is active."

**Comment by quadsYT** [comment]
I believe i have done it

**Comment by ha1vorsen** [comment]
Looks good to me.

**Comment by el-emmo** [comment]
A note that autoRCM is disabled when updating in stock also exists in the `Advantages` section that follows. Do you think we need two of them?

**Comment by quadsYT** [comment]
This pr is possibly useless then..

File: docs/extras/autorcm.md [hunk 1]
```diff
@@ -18,6 +18,7 @@
 	- Please keep in mind that the console can no longer boot on its own, so you'll need a PC, phone, or other payload injector to start the console after letting the console fully shut off. (AutoRCM by itself is considered a "tethered jailbreak").
 		Note: Sleep Mode is not considered "powering off/on" the console. Sleep Mode will still work as intended and is fully unchanged with AutoRCM.
 	- When completely discharged, your Switch will take a very long time to charge while in RCM. To solve this, charge the console for approximately twenty minutes before booting into Hekate and selecting a boot option. After it's done booting, the console will now charge at a normal speed with a battery icon in the corner.
+	-When a system update is completed while stock firmware is running, AutoRCM becomes disabled so you need to turn it on again. AutoRCM can be preserved by updating while CFW is active."
```
Comment (el-emmo) [inline docs/extras/autorcm.md:21]: For consistency, probably add a space after the dash and before the `when`.

**Comment by el-emmo** [comment]
I guess it doesn't hurt to have that note another time, autorcm being disabled after updating can be an issue for many users after all.

File: docs/extras/autorcm.md (hunk 1 above)
Comment (quadsYT) [reply docs/extras/autorcm.md:21]: Alright

**Review by el-emmo — APPROVED** [review]

**Pull request merged**